daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md

```diff
@@ -78,6 +78,10 @@ Dapr applications can subscribe to published topics. Dapr allows two methods by
 
 In principle, Dapr considers message successfully delivered when the subscriber responds with a non-error response after processing the message. For more granular control, Dapr's publish/subscribe API also provides explicit statuses, defined in the response payload, which the subscriber can use to indicate the specific handling instructions to Dapr (e.g. `RETRY` or `DROP`). For more information on message routing read [Dapr publish/subscribe API documentation]({{< ref "pubsub_api.md#provide-routes-for-dapr-to-deliver-topic-events" >}})
 
+### Handling failed messages with dead letter topics
+
+Dapr allows developers to set dead letter topics to deal with messages that cannot be delivered to any receiver. This feature is available on all pub/sub components and prevents consumers from endlessly retrying a failed message. For more information, read [Dead Letter Topics]({{< ref "pubsub-deadletter.md">}})
```

Comment:
```suggestion
Sometimes, messages can't be processed because of a variety of possible issues, such as erroneous conditions within the producer or consumer application or an unexpected state change that causes an issue with your application code. Dapr allows developers to set dead letter topics to deal with messages that cannot be delivered to an application. This feature is available on all pub/sub components and prevents consumer applications from endlessly retrying a failed message. For more information, read about [dead letter topics]({{< ref "pubsub-deadletter.md">}})
```